config/application.config.php

```diff
@@ -1,22 +1,29 @@
 <?php
-return [
-    // This should be an array of module namespaces used in the application.
-    'modules' => [
-        'ZendDeveloperTools',
-        'DoctrineModule',
-        'DoctrineORMModule',
-        'User',
-        'Application',
-        'Company',
-        'Decision',
-        'Frontpage',
-        'Activity',
-        'Photo',
-        'Education',
-        'Migration',
-        'MaglMarkdown'
-    ],
+$env = getenv('APP_ENV') ?: 'production';
```

Comment:
The configuration for the module should depend on the environment variables, and the other code (ideally) should not. There are several ways to do this. Either by setting environment related variables in .env files (or a mechanism that does the same, like the .local files). This is especially a good solution for passwords etc. Or different config files need to be loaded according to the environment. I.e. you have a common config file, and a config file for production, development, staging and testing. You then merge the common config file, with the environment specific file.